setup.py

```diff
@@ -54,7 +54,6 @@
     "reana-db>=0.8.0a1,<0.9.0",
     "htcondor==8.9.7",
     "retrying>=1.3.3",
-    "paramiko[gssapi]>=2.6.0",
```

Comment:
We cannot remove `paramiko` as a dependency otherwise the REANA-Job-Controller package will stop working [when it tries to use this library](https://github.com/reanahub/reana-job-controller/blob/5ac0d25ee128db8bbd51fe74f01ebfa11620a34e/reana_job_controller/utils.py#L83-L84).

Isn't it enough to include `autodoc_mock_imports` as you have done? Maybe we just need to find all the packages that make the build fail and add them to this list?

Reply:
If I keep `paramiko[gssapi]` in `setup.py`, it gives an error because it tries to first install `gssapi`, but It works if I just keep `paramiko`.

Reply:
We need to make it pass without modifying the content of `setup.py`. The current content of `setup.py` is necessary for REANA-Job-Controller to work correctly, therefore, we must find a solution elsewhere.